src/economics.js

```diff
@@ -181,6 +181,30 @@ var eco = {
                 logr.econ(newCoins + ' dist from the vote')
                 logr.econ(newBurn + ' burn from the vote')
 
+                // compute final claimable amount after author tip
+                let authorVote = -1
+                let authorVoteClaimed = false
+                let totalAuthorTip = 0
+                let precisionMulti = Math.pow(10,config.ecoClaimPrecision+config.tippedVotePrecision)
+                for (let v = 0; v < newVotes.length; v++)
+                    if (newVotes[v].u === content.author) {
+                        authorVote = v
+                        if (newVotes[v].claimed) authorVoteClaimed = true
```

Comment:
Wouldn't be better, performance-wise, to 'break' the loop here?
So we don't iterate over all the votes if we have already found the author.
Imagine doing that for every vote on a video with millions of votes.

Reply:
The break is already [here](https://github.com/dtube/avalon/compare/master...techcoderx:tipped-vote#diff-dcfff47c0bb294c2fefa6290af5da5d809aa50251af8883d5dfbcdeb05fa9865R193), you were probably looking at the wrong commit

Reply:
Ok, I didn't see that one, that's fine.